reference/errors/error_codes.mdx

```diff
@@ -423,6 +431,10 @@ Meilisearch could not find the target document. Make sure your target document i
 
 [`attributesToRetrieve`](/reference/api/search#attributes-to-retrieve) is invalid. It should be an array of strings, a string, or set to null.
 
+### `invalid_similar_embedder`
+
+[`embedder`](/reference/api/search#hybrid-search) is invalid. It should be a string corresponding to the name of a configured embedder.
```

Comment:
the `embedder` link should point to the version of the parameter in the similar route documentation, should it not?